github/gh-collaboration.qmd

```diff
@@ -23,4 +23,6 @@
 
 ## Making changes
 
-## Merging
+## Teacher merges the pull request
+
+- Once both teacher and learner are satisfied with the updates, the teacher merges the pull request.

```

Comment:
One thing to clarify is that the teacher has a few things to merge.
- The learner's PR into their lesson branch
- And once everything is received, the lesson branch into the main branch